.vscode/settings.json

```diff
@@ -19,21 +17,17 @@
   // "explorer.confirmDelete": false,
   // "explorer.confirmDragAndDrop": false,
 
-  "eslint.enable": true,
   "eslint.runtime": "node",
   "eslint.format.enable": true,
-  "eslint.alwaysShowStatus": true,
-  "eslint.workingDirectories": [ { "mode": "auto" } ],
+  "eslint.workingDirectories": [{ "mode": "auto" }],
   // "eslint.packageManager": "npm",
-  "eslint.validate": ["javascript","javascriptreact","typescript","typescriptreact"],
-  "editor.defaultFormatter": "dbaeumer.vscode-eslint",
```

Comment:
`dbaeumer.vscode-eslint` was wrongly configured as formatter, but it is not a formatter but a linter and threw a warning inside VSCode. Now it is corrected to be Prettier instead.